src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -184,6 +188,21 @@ def _task_format_group(item):
     ])
 
 
+def _taskrun_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('RUN ID', _get_value(item, 'runId')),
+        ('TASK', _get_value(item, 'task')),
+        ('PLATFORM', _get_value(item, 'platform', 'os')),
+        ('STATUS', _get_value(item, 'status')),
+        ("TRIGGER", _get_build_trigger(_get_value(item, 'imageUpdateTrigger'),
+                                       _get_value(item, 'sourceTrigger', 'eventType'),
+                                       _get_value(item, 'timerTrigger'))),
```

Comment:
The trigger property is not very relevant to taskrun as it's always triggered in ARM deployment in normal case.